tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,441 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "photlam_hatp11 = SourceSpectrum(Empirical1D,\n",
+    "                                points=wavelengths, lookup_table=flux)\n",
+    "\n",
+    "# Compute the flux at Earth from the luminosity \n",
+    "R = 0.683 * u.R_sun  # radius in solar units, Deming et al. 2011\n",
+    "d = 123 * u.lyr # distance to HATP11 in light years from Gaia parallax\n",
+    "photlam_hatp11 = photlam_hatp11 * float(R / d) ** 2 / np.pi\n",
```

Comment:
If/when we figure out this extra factor of pi, we should leave a comment explaining why it's necessary.